dwm.c

```diff
@@ -3270,7 +3270,7 @@ void
 updatestatus(void)
 {
 	if (!gettextprop(root, XA_WM_NAME, stext, sizeof(stext))) {
-		strcpy(stext, "dwm-titus:"VERSION);
```

Comment:
Thanks 🙃

Reply:
There's absolutely no reason to display dwm-titus in the status bar when you could just stick with dwm. So, yeah, I'm merely trying to maintain simplicity.